### ISSUE TITLE
Lumina: unregistered desktop

### ISSUE THREAD

**Issue by rezso** [pcbsd/lumina#49]
Output of desktop-file-validate command:
lumina-info.desktop: error: value "Lumina" for key "OnlyShowIn" in group "Desktop Entry" contains an unregistered value "Lumina"; values extending the format should start with "X-"
The pcbsd uses a patched desktop-file-utils, or this is a bug?

**Comment by beanpole135** [comment]
The "desktop-file-validate" utility you refer to probably is not aware of the Lumina desktop (this was also an issue for a while with users of "xdg-open" and other cross-DE utilities). 

However, I did just find a capitalization issue with the lumina-info.desktop file, where is set "OnlyShowIn=Lumina", but the XDG_CURRENT_DESKTOP value is set to "LUMINA". I just fixed that in the source code a moment ago.

Link to the standards governing the use of the this file:
http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s05.html
